app/models/user.rb

```diff
@@ -38,4 +38,8 @@ def notes
   def select_status_list
     VALID_STATUS_LIST.map { |status| [I18n.t("user.status.#{status}"), status] }
   end
+
+  def name
```

Comment:
even if this is simple, I think there should be tests for all methods in the model